public/js/office.web.js

```diff
@@ -130,42 +126,53 @@ $(() => {
     };
   }
 
-  function startVideoConference(roomId, name, socket){
-    $("#exampleModalCenter").modal("hide");
-    $("#exampleModalCenter").modal("dispose");
-  	const domain = 'meet.jit.si';
-		const options = getMeetingOptions(roomId);
-		api = new JitsiMeetExternalAPI(domain, options);
-		api.executeCommand('displayName', matrixProfile.loadStoredProfile().name);
-		api.executeCommand('avatarUrl', matrixProfile.loadStoredProfile().imageUrl);
-		$("#exampleModalCenter").modal("show");
-
-    socket.emit('start-meet', matrixProfile.loadStoredProfile().id);
-
-		$("#exampleModalCenter").on("hidden.bs.modal", function () {
-        socket.emit('left-meet', matrixProfile.loadStoredProfile().id);
-   			api.dispose();
-    });
-
-    $('#exampleModalCenter').on('shown.bs.modal', function () {
-      var modal = $(this);
-      modal.find('.modal-title').text(name);
-    });
+  function isUserInVideoConference(){
+    if($("#exampleModalCenter").data('bs.modal')){
+      return $("#exampleModalCenter").data('bs.modal')._isShown;  
+    }
+    return false;
   }
 
-  function saveLastRoom(data) {
-    localStorage.setItem(`last_room${data.user.id}`, data.room);
+  function startVideoConference(roomId, name, officeEvents){
+    setTimeout(() => {
+      $("#exampleModalCenter").modal("hide");
+      $("#exampleModalCenter").modal("dispose");
+    	const domain = 'meet.jit.si';
+  		const options = getMeetingOptions(roomId);
+  		api = new JitsiMeetExternalAPI(domain, options);
+  		api.executeCommand('displayName', matrixProfile.loadStoredProfile().name);
+  		api.executeCommand('avatarUrl', matrixProfile.loadStoredProfile().imageUrl);
+  		
+      officeEvents.startMeet();
+
+      $("#exampleModalCenter").modal("show");
+  		$("#exampleModalCenter").on("hidden.bs.modal", function () {
+          officeEvents.leftMeet();
+     			api.dispose();
+      });
+
+      $('#exampleModalCenter').on('shown.bs.modal', function () {
+        var modal = $(this);
+        modal.find('.modal-title').text(name);
+      });
+    }, 300);  
   }
 
-  function notify(data, title) {
+  function notifyRoomEnter(user,roomId) {
     const options = {
-      icon: data.user.imageUrl,
+      icon: user.imageUrl,
     };
 
     if (Notification.permission !== 'granted') {
       Notification.requestPermission();
     } else {
-      const n = new Notification(title, options);
+      const loggedUserId = matrixProfile.loadStoredProfile().id;
+      const loggedUserRoomId = getLastRoom(matrixProfile);
+
+      if (loggedUserRoomId == roomId && loggedUserId != user.id) {
+        const roomTitle = getRoomName(roomId);
+        const n = new Notification(`${user.name} entered into the room ${roomTitle}`, options);   
```

Comment:
Why do you need to attribute the notification instance to a variable ?

Reply:
Opssss vou arrumar..hehehe